controllers/AuthController.js

```diff
@@ -211,6 +213,7 @@ export default class AuthController {
         },
       });
     } catch (error) {
+      /* istanbul ignore next-line */
```

Comment:
Why don't you want this line covered?

Reply:
It's the social login feature, my feature is properly covered but passport and that line was dropping my coverage so ignored it. i didn't collaborate fully with him on that feature so i don't know what to test for.